### DISCUSSION TITLE
这软件到底是收费还是免费？

### DISCUSSION THREAD

**Discussion by mtueih** [d2phap/ImageGlass#1993] [General] [answered]
要是收费，就别tm挂开源免费的标签，也tm别挂到github上。要是普通版免费，专业版收费，那就更别挂github，弄个官网，把收费情况说清楚。不知道你官网挂着开源免费的标签，并且在github上也有仓库，又以收费形式上架微软商店，居心何在？你收费就收费，开源就开源，免费就免费，别tmd挂羊头卖狗肉，你收费你就干脆点，妈的老子又不是买不起，你敢卖老子就敢买啊，你tmd既然开源免费，在微软商店收费是几个意思？还有开源软件都要收费，那算真的开源吗？你真的以为做个这个软件有多了不起啊，这么玩弄人心，这样自欺欺人？要收费，就被搞什么开源免费的标签，也离github远一点，也别搞什么免费开源了，老子想到微软商店这个软件收费的摸样，老子就nm想吐，这样的软件背后的人，真tm让我觉得恶心，妈的竟然还是github上的，你挂到github挂个jb啊，去圈nm的钱去吧，别沾上开源免费的标签，真是玷污了开源精神

**Comment by d2phap** [comment]
English or Vietnamese only

**Reply by mtueih** [reply]
If you're going to charge for it, then don't fucking label it as "open source and free," and don't fucking put it on GitHub either. If the standard version is free and the Pro version is paid, then all the more reason not to put it on GitHub. Set up an official website instead and clearly explain what's free and what people have to pay for.

I honestly don't know what you're trying to achieve. You've got an "open source and free" label on your official website, you've got a GitHub repository, and at the same time you're selling it on the Microsoft Store. What the fuck is that supposed to mean?

If you're charging for it, then just charge for it. If it's open source, then just make it open source. If it's free, then just make it free. Don't fucking advertise one thing and sell another.

If you're going to charge people, then just be upfront about it. Damn it, it's not like I can't afford it. If you've got the guts to sell it, I've got the money to buy it. But if you're claiming it's open source and free, then what the hell is the point of charging for it on the Microsoft Store?

And another thing: if open-source software has to be paid for, is it really open source? Do you seriously think building a piece of software like this makes you that special? Are you really fooling people like this while lying to yourself at the same time?

If you're going to charge money, then stop calling it "open source and free." Stay the hell away from GitHub, too. Don't pretend it's free and open source anymore.

Just thinking about the fact that this software is being sold on the Microsoft Store makes me want to fucking puke. The kind of people behind software like this honestly make me sick.

And the fact that it's even on GitHub? What the fuck are you doing putting it there? Go make your fucking money somewhere else, but don't drag the "open source and free" label into it. You're nothing but a disgrace to the spirit of open source.

**Reply by d2phap** [reply]
I don't know and don't care what happened to you but you deserve to get blocked 😉

**Comment by zhengsehelp** [comment]
😓

**Comment by louhao1129** [comment, marked as answer]
免费的，查看下面的链接可以下载

https://github.com/d2phap/ImageGlass/releases

**Comment by mefly521** [comment]
别从windows 应用商店装，那个是收费的
